<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GotoDialog</class>
 <widget class="QWidget" name="GotoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>160</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>转到指定行</string>
  </property>
  <widget class="QLineEdit" name="gotoEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>55</y>
     <width>261</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 3px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>25</y>
     <width>108</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>行号(L):</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>110</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 6px;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gotoButton">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>110</y>
     <width>90</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 6px;</string>
   </property>
   <property name="text">
    <string>转到</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
